sale_operating_unit/README.rst

```diff
@@ -13,21 +13,23 @@ Operating Unit in Sales
 .. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
     :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
     :alt: License: LGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Foperating--unit-lightgray.png?logo=github
-    :target: https://github.com/OCA/operating-unit/tree/13.0/sale_operating_unit
-    :alt: OCA/operating-unit
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--auth-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-auth/tree/13.0/sale_operating_unit
```

Comment:
Check the command that generates README is pointing to operating-unit repository